stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/resources/ConnectorNodeCredentialConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.ida.notification.stubconnector.resources;
```

Comment:
Is this the correct place for this? Perhaps somewhere like `uk.gov.ida.notification.stubconnector.configuration`

Reply:
Agree! Moving